src/sv-pipeline/scripts/downstream_analysis_and_filtering/apply_minGQ_filter.py

```diff
@@ -414,7 +417,11 @@ def main():
             for key in 'AN AC AF N_BI_GENOS N_HOMREF N_HET N_HOMALT FREQ_HOMREF FREQ_HET FREQ_HOMALT'.split(' '):
                 if key in record.info.keys():
                     record.info.pop(key)
-        
+
+        # Standardize SVTYPE for all INS variants, if optioned
+        if args.simplify_INS_SVTYPEs and 'INS' in record.info['SVTYPE']:
+            record.info['SVTYPE'] = 'INS'
```

Comment:
Might be better to do `and 'INS' in record.info['SVTYPE'].split(':')` - just a suggestion